docs/machine-learning/tutorials/object-detection-model-builder.md

```diff
@@ -0,0 +1,261 @@
+---
+title: 'Tutorial: Detect objects in images with Model Builder'
+description: This tutorial illustrates how to build an object detection model using ML.NET Model Builder and Azure ML to detect stop signs in images.
+author: briacht
+ms.author: brachtma
+ms.date: 3/12/2021
+ms.topic: tutorial
+ms.custom: mlnet-tooling
+#Customer intent: As a non-developer, I want to use Model Builder to automatically generate a model to detect stop signs in images using Model Builder.
+---
+
+# Tutorial: Detect stop signs in images with Model Builder
+
+Learn how to build an object detection model using ML.NET Model Builder and Azure ML to detect and locate stop signs in images.
+
+In this tutorial, you learn how to:
+
+> [!div class="checklist"]
+>
+> - Prepare and understand the data
+> - Choose the scenario
+> - Choose the training environment
+> - Load the data
+> - Train the model
+> - Evaluate the model
+> - Use the model for predictions
+
+> [!NOTE]
+> Model Builder is currently in Preview.
+
+## Prerequisites
+
+For a list of prerequisites and installation instructions, visit the [Model Builder installation guide](../how-to-guides/install-model-builder.md).
+
+## Model Builder object detection overview
+
+Object detection is a computer vision problem. While closely related to image classification, object detection performs image classification at a more granular scale. Object detection both locates _and_ categorizes entities within images. Use object detection when images contain multiple objects of different types.
+
+![Screenshots showing Image Classification versus Object Classification.](./media/object-detection-onnx/img-classification-obj-detection.png)
+
+Some use cases for object detection include:
+
+- Self-Driving Cars
+- Robotics
+- Face Detection
+- Workplace Safety
+- Object Counting
+- Activity Recognition
+
+This sample creates a C# .NET Core console application that detects stop signs in images using a machine learning model built with Model Builder. You can find the source code for this tutorial at the [dotnet/machinelearning-samples](https://github.com/dotnet/machinelearning-samples/tree/master/samples/modelbuilder/ObjectDetection_StopSigns) GitHub repository.
```

Comment:
Can you add it to the samples browser and use the samples browser link instead?

Reply:
What do you mean by samples browser?

Reply:
See https://docs.microsoft.com/en-us/teamblog/code-samples

See also https://github.com/dotnet/machinelearning-samples/blob/0f01ae23a204d142923fb678c5f178b10592674d/samples/modelbuilder/ImageClassification_Azure_LandUse/README.md for an example from marchinelearning-samples repository, which is published in https://docs.microsoft.com/en-us/samples/dotnet/machinelearning-samples/mlnet-image-classification-transfer-learning/ because of the special YAML metadata in the README.md.

Reply:
How do I add to the samples browser?

Reply:
@Youssef1313 I'm not sure how to add to the samples browser

Reply:
@briacht By adding a README.md in the sample directory in machinelearning-samples repo. The README.md should have the following at the top:

```csharp
---
page_type: sample
name: "Sample name"
description: "A small description"
urlFragment: "this-will-be-part-of-the-url-when-sample-is-published"
languages:
- csharp
products:
- dotnet
- dotnet-core
---
```

Reply:
Gotcha thanks! Added with https://github.com/dotnet/machinelearning-samples/pull/882/commits/4da24f944b4ce54b85bf77221d8ff762c4f090ad